drivers/hmis_csv_importer/app/models/hmis_csv_importer/hmis_csv_validation/unique_hud_key.rb

```diff
@@ -13,6 +15,8 @@ def self.check_validity!(klass, importer_log)
       group(klass.hud_key).
       having(nf('COUNT', [klass.hud_key]).gt(1)).
       count
+
+    return [] if incorrect_counts.is_a?(Integer) && incorrect_counts.zero?
```

Comment:
I guess this can be nil somehow? Probably clearer to say:

```suggestion
    return [] if incorrect_counts&.zero?
```

Reply:
This weird check is a result of originally having what you suggest.  Sometimes it appears to return zero, sometimes it returns an array...